services/qingstor/storager_test.go

```diff
@@ -7,16 +7,17 @@ import (
 	"testing"
 	"time"
 
-	"github.com/Xuanwo/storage/pkg/storageclass"
-	"github.com/Xuanwo/storage/services"
-	"github.com/Xuanwo/storage/types/metadata"
 	"github.com/golang/mock/gomock"
 	"github.com/google/uuid"
 	"github.com/pengsrc/go-shared/convert"
 	"github.com/stretchr/testify/assert"
 	qerror "github.com/yunify/qingstor-sdk-go/v3/request/errors"
 	"github.com/yunify/qingstor-sdk-go/v3/service"
 
+	"github.com/Xuanwo/storage/pkg/storageclass"
+	"github.com/Xuanwo/storage/services"
+	"github.com/Xuanwo/storage/types/metadata"
+
```

Comment:
remove this empty line.

Reply:
These are current project imports, so it is preferred to put them into a single group.